frontend/public/style/_common.scss

```diff
@@ -172,18 +162,6 @@ dl.co-inline {
   margin-bottom: 30px;
 }
 
-.co-m-pane__details {
```

Comment:
Need to document removal as [other repos are affected](https://github.com/search?q=org%3Aopenshift%20co-m-pane__details&type=code).